LICENSE-binary

```diff
@@ -0,0 +1,292 @@
+
+   Copyright [yyyy] [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+------------------------------------------------------------------------------------
+This project bundles some components that are also licensed under the Apache
+License Version 2.0:
+
+org.apache.commons:commons-lang3
+org.apache.tez:tez-api
+org.apache.tez:tez-common
+org.apache.tez:tez-dag
+org.apache.tez:tez-runtime-internals
+org.apache.tez:tez-runtime-library
+org.apache.tez:tez-tests
+org.eclipse.jetty:jetty-server
+org.eclipse.jetty:jetty-servlet
+org.eclipse.jetty:jetty-util
+commons-cli:commons-cli
+org.apache.hbase.thirdparty:hbase-shaded-jersey
+org.apache.directory.api:api-asn1-api
+org.apache.httpcomponents:httpclient
+net.jpountz.lz4:lz4
+com.google.guava:guava
+com.fasterxml.jackson.core:jackson-annotations
+com.fasterxml.jackson.core:jackson-core
+com.fasterxml.jackson.core:jackson-databind
+com.fasterxml.jackson.module:jackson-module-scala_2.12
+info.picocli:picocli
+org.apache.directory.server:apacheds-kerberos-codec
+org.apache.hadoop:hadoop-common
+org.apache.hadoop:hadoop-mapreduce-client-app
+org.apache.hadoop:hadoop-mapreduce-client-core
+org.apache.hadoop:hadoop-mapreduce-examples
+org.apache.hadoop:hadoop-minicluster
+org.apache.hadoop:hadoop-minikdc
+org.apache.hadoop:hadoop-yarn-api
+org.apache.hadoop:hadoop-yarn-server-common
+com.google.errorprone:error_prone_annotations
+io.netty:netty-all
+org.xerial.snappy:snappy-java
+io.grpc:grpc-netty-shaded
+io.grpc:grpc-protobuf
+io.grpc:grpc-stub
+io.grpc:grpc-testing
+org.awaitility:awaitility
+io.prometheus:simpleclient
+io.prometheus:simpleclient_hotspot
+io.prometheus:simpleclient_httpserver
+io.prometheus:simpleclient_jetty
+io.prometheus:simpleclient_pushgateway
+io.prometheus:simpleclient_servlet
+org.apache.spark:spark-core_2.12
+org.apache.spark:spark-sql_2.12
+org.roaringbitmap:RoaringBitmap
+javax.inject:javax.inject
+
+-----------------------------------------------------------------------------------
+This product bundles various third-party components under other open source licenses.
+This section summarizes those components and their licenses. See licenses-binary/
+for text of these licenses.
+
+BSD 2-Clause
+------------
+com.github.luben:zstd-jni
+
+BSD 3-Clause
+------------
+com.google.protobuf:protobuf-java
+com.google.protobuf:protobuf-java-util
+
+MIT License
+-----------
+org.slf4j:slf4j-api
+org.slf4j:slf4j-reload4j
+org.mockito:mockito-core
```

Comment:
Likewise, if mockito is only used for testing, it should be possible to omit it from this file.